gulp/tasks/gen-index-html.js

```diff
@@ -15,16 +16,26 @@ function generateIndex(es6) {
   const targetUrl = es6 ? '/frontend_latest/' : '/frontend_es5/';
 
   const toReplace = [
+    // Needs to look like a color during CSS minifiaction
+    ['{{ theme_color }}', '#THEME'],
     ['/home-assistant-polymer/hass_frontend/mdi.html',
       `/static/mdi-${md5(path.resolve(config.output, 'mdi.html'))}.html`],
-    ['/home-assistant-polymer/build-temp-es5/compatibility.js',
-      `/static/compatibility-${md5(path.resolve(config.output_es5, 'compatibility.js'))}.js`],
   ];
 
   if (!es6) {
     toReplace.push([
       '/service_worker.js', '/service_worker_es5.js'
     ]);
+
+    const compatibilityPath = `/frontend_es5/compatibility-${md5(path.resolve(config.output_es5, 'compatibility.js'))}.js`;
+    const es5Extra = `
+    <script src='${compatibilityPath}'></script>
+    <script src='/frontend_es5/custom-elements-es5-adapter.js'></script>
```

Comment:
Does minifier inlines those 2 scripts?
It would be great if it did.

Reply:
I can add that. Should we inline core.js too?

Reply:
There are pros and cons:

On cold start it will save a round trip.
However index.html is fetched each time time to update cache, so by increasing this file we waste traffic.

Another option is to unite core+compatibility+adapter to a single file.

Reply:
Just realized that we can't combine `custom-elements-es5-adapter.js` with anything else. It will contain invalid syntax for older browsers which stops execution of the rest of the file.